.github/scripts/auto-backport.py

```diff
@@ -123,7 +114,7 @@ def backport(repo, pr, version, commits, backport_base_branch):
                     repo_local.git.cherry_pick('--continue')
             repo_local.git.push(fork_repo, new_branch_name, force=True)
             create_pull_request(repo, new_branch_name, backport_base_branch, pr, backport_pr_title, commits,
-                                is_draft=is_draft)
+                                is_draft=is_draft, is_collaborator=is_collaborator)
```

Comment:
nit, the kwargs style is not necessary here. as the names of the arguments are self-explanatory.